src/styles/BookList.module.css

```diff
@@ -0,0 +1,180 @@
+/* For BookList.jsx */
+.booklist {
+  background-color: #fafafa;
+  display: flex;
+  flex-direction: column;
+  margin: 2% 9%;
+}
+
+/* For Book.jsx */
+.bookContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  border-radius: 4px;
+  border: 1px solid #e8e8e8;
+  margin: 2%;
+  background-color: #fff;
+  padding: 1%;
+}
+
+.bookInnerContainer {
+  display: flex;
+  flex-direction: column;
+  margin: 2% 0;
+}
+
+.bookButtons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.divider {
+  border: 1px solid #e0e0e0;
+  margin: 0 5%;
+}
+
+/* // bookButtons should be just text buttons in color blue. */
+.bookButton {
+  color: #0290ff;
+  font-family: Montserrat, sans-serif;
+
+  /* font-size: 0.813rem; */
```

Comment:
## [OPTIONAL]

- [ ] Kindly remove commented code as they will make your code clean and clear